playbooks/roles/ocp-cli-plugins/README.md

```diff
@@ -0,0 +1,94 @@
+OCP CLI-plug-ins deployment and testing
+=======================================
+
+The role *"roles/ocp-cli-plugins"* provides automated deployment and testing for the CLI-plugins oc-mirror, oc-compliance, krew-cli and custom-cli plugin. User can add CLI-plugin names in the role variable *deploy_cli_plugins* for deploy and test the plugins as explained in **Role Variables** section.
+This role install the prerequisite tools and depedencies for these plugins before deploying them.
+
+
+Requirements
+------------
+
+ - Running OCP 4.x cluster.
+
+
+Role Variables
+--------------
+
+| Variable                       | Required | Default                                                                                                         |Comments                                                                                                                                                                                                                            |
+|--------------------------------|----------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| enable_cli_plugins_test        | no       | false                                                                                                           | Flag to be set to true to enable custom CLI-plug-in test playbook                                                                                                                                                                  |
+| deploy_cli_plugins             | yes      | empty list                                                                                                      | List of cli-plugin names to be deployed and tested, plugin names must be among ["custom","krew","mirror","compliance"]. Empty list will not deploy or test any cli-plugin                                                          |
+| user_custom_plugin             | no       | empty string                                                                                                    | File name of user defined custom-cli-plugin prefixed with "oc-" e.g. user_custom_plugin: "oc-test-plugin.txt"                                                                                                                      |
```

Comment:
is there a template available that can be added here for users to use?

Reply:
There is a test plugin `role_path/files/oc-test-plugin.sh` in role files which can be referred, added the comment to `user_custom_plugin` role variable for same.